notebooks/samples/optimizer/ai_platform_optimizer_conditional_parameters.ipynb

```diff
@@ -79,7 +79,7 @@
         "id": "VQ3-QAet1Ufd"
```

Comment:
<p>Can you separate this command into a new cell ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>